tests/framework/Databases/HDF5/tests

```diff
@@ -59,6 +59,11 @@
     heavy = true
   [../]
 
+  [./hdf5_long_array_strings]
+    type = 'RavenFramework'
+    input = 'test_hdf5_long_array_strings.xml'
+    output = 'HDF5_with_strings/reprint_hs.csv'
```

Comment:
should we check the csv values?

Reply:
I think this should be enough because it is aimed to test the mechanics (no crash).

But If you want me to add the gold file, I will